tests/test_userspace.py

```diff
@@ -91,6 +91,23 @@ def test_hostname(shell, check):
         assert serial_number in etc_hostname
 
 
+def test_system_degraded(shell):
+    """
+    Test if the system state is degraded.
+    """
+
+    state = shell.run_check("SYSTEMD_COLORS=0 systemctl status --no-pager")
```

Comment:
I _think_ you can get the same information from `systemctl is-system-running`:

```
leonard@fulladder ~> systemctl is-system-running
running
```

That would mean no parsing.